docs/standard/exceptions/best-practices-for-handling-and-using-exceptions-outline.md

```diff
@@ -0,0 +1,91 @@
+---
+title: [Best practices for handling and using exceptions]
+description: [Practical guide to handling and using exceptions.]
+author: [sam.wheat@outlook.com]
+ms.date: [12/10/2018]
+---
+# Best practices for handling and using exceptions
+
+@BillWagner For your review 
+
+1.	An approach to using exceptions and exception handling to your advantage.
+	1.	Start with reasonable expectations.

```

Comment:
This section might change based on my comment on exceptions outside of a programmer's control

Reply:
The goal here is the same as your assertion in the comment below:  Exceptions are often unavoidable, proceeding blindly and sidestepping them is not a good choice, etc.